indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -1157,7 +1160,9 @@ private void maybePersistAndPublishSequences(Supplier<Committer> committerSuppli
   {
     for (SequenceMetadata<PartitionIdType, SequenceOffsetType> sequenceMetadata : sequences) {
       sequenceMetadata.updateAssignments(currOffsets, this::isMoreToReadBeforeReadingRecord);
-      if (!sequenceMetadata.isOpen() && !publishingSequences.contains(sequenceMetadata.getSequenceName())) {
+      if (!sequenceMetadata.isOpen() &&
+          !publishingSequences.contains(sequenceMetadata.getSequenceName())
```

Comment:
Style: symmetry with the next condition.

```suggestion
      if (!sequenceMetadata.isOpen()
          && !publishingSequences.contains(sequenceMetadata.getSequenceName())
```

Reply:
Done